samples/payload/Makefile

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+##
+##  Makefile for Detours Test Programs.
+##
+##  Microsoft Research Detours Package
+##
+##  Copyright (c) Microsoft Corporation.  All rights reserved.
+##
+
+!include ..\common.mak
+
+CFLAGS=$(CFLAGS) /std:c++17 /EHsc
```

Comment:
People routinely report and fix issues compiling detours with MSVC 2005 and other crazy old configurations, how do I'm worried about taking a dependency on modenish c++, I agree it would be nice but not sure how realistic it is.

Reply:
I added that for cppwinrt but I'll remove it.

Reply:
Should work with old compilers now